pages/_language/events/_event/index.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-  <div>
-    <header>
+  <div class="event">
+    <header class="event__layout">
```

Comment:
`event__layout` -> `event__section`?

Reply:
Want to refactor this because we're going to re-use this styling on the event chapter page that's why I named it differently.

Currently there are layout classes which are too specific to use but would be 'the best solution'. Discussed with Bas if it's worth changing this core styling, it will be a difficult change but we might want to...